sdk/src/types/block/core/wrapper.rs

```diff
@@ -128,6 +128,30 @@ impl BlockWrapper {
         &self.signature
     }
 
+    /// Returns the work score of a [`BlockWrapper`].
```

Comment:
Maybe add "in kilobytes" to the comment

Reply:
Honestly I think documenting `in kilobytes` might be a little bit confusing. Yes, the size is taken into account, but also other things that aren't byte size related. So I wouldn't add it tbh.